Formula/fluid-synth.rb

```diff
@@ -29,11 +34,18 @@ def install
 
     mkdir "build" do
       system "cmake", "..", *args
+      system "make", "check"
```

Comment:
Everything else is OK, but please remove `make check`: we don't generally run compile-time tests, except for security-related formulas.

Reply:
Alright; I'm pushing the change now. I saw it in other formulae (probably the security-related ones you are talking about) and was curious but didn't get a response in the other PR. Thanks!